lib/mappers/bhl_mods.py

```diff
@@ -42,17 +41,14 @@ def map_description(self):
         if exists(self.provider_data, prop):
             description = []
             for s in iterify(getprop(self.provider_data, prop)):
-                try:
-                    if s.get("type") == "content":
-                        description.append(s.get("#text"))
-                except:
-                    logger.error("Error getting note/type from record %s" %
-                                 self.provider_data["_id"])
+                if "type" in s and s.get("type") == "content":
+                    description.append(s.get("#text"))
 
-            self.update_source_resource({"description": description})
+            if description:
+                self.update_source_resource({"description": description})
 
-    # OAI MODS Mapper is blowing up on this method because of arg c
-    # ount issues, even though we handle it below
+    # OAI MODS Mapper is blowing up on this method because of arg
+    # count issues,
```

Comment:
I think with my comment here, I'm not necessarily noting the lack of a mapping (`format` seems to be assigned in `map_format_and_spec_type`), but instead a misalignment in the number of arguments for the `map_format` function going up the inheritance chain. The Mapper class has a no-args version, while OAIMODSMapper adds an `authority_condition` parameter. Without adding in the no-args version of `map_format,` an exception is thrown when this class is actually used.

I didn't spend a lot of time investigating this because we're looking at replacing this stuff in the new year.